modules/ballerina-core/src/main/java/org/ballerinalang/util/debugger/util/MessageUtil.java

```diff
@@ -0,0 +1,235 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing,
+*  software distributed under the License is distributed on an
+*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*  KIND, either express or implied.  See the License for the
+*  specific language governing permissions and limitations
+*  under the License.
+*/
+package org.ballerinalang.util.debugger.util;
+
+import org.ballerinalang.model.util.JsonGenerator;
+import org.ballerinalang.model.util.JsonNode;
+import org.ballerinalang.model.util.JsonParser;
+import org.ballerinalang.util.debugger.DebugConstants;
+import org.ballerinalang.util.debugger.dto.BreakPointDTO;
+import org.ballerinalang.util.debugger.dto.CommandDTO;
+import org.ballerinalang.util.debugger.dto.FrameDTO;
+import org.ballerinalang.util.debugger.dto.MessageDTO;
+import org.ballerinalang.util.debugger.dto.VariableDTO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Contains util methods to convert data objects to json and vise versa.
+ *
+ * @since 0.95.5
+ */
+public class MessageUtil {
```

Comment:
Shall we rename this class? I think it doesn't relate to "message" type?

Reply:
"message" type is not there anymore, anyway renamed it to "DebugMsgUtil"